scripts/mobile/android-repack-asan.sh

```diff
@@ -52,6 +52,7 @@ DEFOLD_HOME="$(cd "${DYNAMO_HOME}/../.."; pwd)"
 
 ANDROID_NDK_VERSION=20
 ANDROID_BUILD_TOOLS_VERSION="32.0.0"
+PLATFORM="darwin-x86_64"
```

Comment:
Not neseccarily the case.
Should work with Linux to I think.

E.g. use something like this:
```
HOST=`uname | tr '[:upper:]' '[:lower:]'`

TOOLHOSTNAME="linux-x86_64"
if [ "$HOST" == "darwin" ]; then
    TOOLHOSTNAME="darwin-x86_64"
fi
```